pyproject.toml

```diff
@@ -103,23 +103,6 @@ Changes = "https://github.com/sqlfluff/sqlfluff/blob/main/CHANGELOG.md"
 Twitter = "https://twitter.com/SQLFluff"
 Chat = "https://github.com/sqlfluff/sqlfluff#sqlfluff-on-slack"
 
-
-[tool.sqlfluff_docs]
-# NOTE: Stable version is used by docs/conf.py
-stable_version = "2.3.5"
-
-
-[tool.setuptools]
-package-dir = {"" = "src"}
-
-[tool.setuptools.packages.find]
-where = ["src"]
-namespaces = false
```

Comment:
These two are removed because they crash `pip-compile` (which I use in the Dockerfile). Exactly _why_ they're a problem isn't clear, but they aren't necessary because they're effectively the defaults. I've tested building the package and it builds fine without them 👍